pandas/core/series.py

```diff
@@ -2970,13 +2970,49 @@ def notnull(self):
 
     def dropna(self, axis=0, inplace=False, **kwargs):
         """
-        Return Series without null values
+        Return Series without null values.
+
+        In case of missing data, any or all values can be used reciprocally.
+        All of missing inputs will be credited.
+
+        Parameters
+        ----------
+        axis : {0 or 'index', 1 or 'columns'}, or tuple/list thereof
+            Pass tuple or list to drop on multiple axes.
```

Comment:
This method is series-specific, so note that \`axis\`  is included for compatability with `DataFrame.dropna`.